### ISSUE TITLE
Incentive Engineering :)

### ISSUE THREAD

**Issue by michael-spengler** [yoshixmk/deno-x-ranking#6]
Hi. Just gave this repo a star :) 

However I wonder if we could come up with even more sophisticated rankings. 

Background: IMO Deno Modules shall be pretty fine granular. I observe people writing many great fine granular Deno modules which get few or no stars. 

Currently out of 942 deno modules more than 600 have 2 or less stars. 
The problem is similar with the other "ranking  data" you listed.

So I wonder what would be event better additional data for rankings. The number of downloads is probably too easy to tweak.

So I do not have a solution proposal yet. Rather I'd like to start a common exploration :)

**Comment by yoshixmk** [comment]
@michael-spengler Thanks a lot :+1:

I think that measuring the value with only star is a problem too.

If we can get information on the number of downloads, it is ideal to reflect it in the ranking, and I consider to collect data historically.

However, Deno has the feature that it can be downloaded and executed from **anywhere in the world** using a URL, which is annoying. Therefore, a **sampling target** is required.

The best way is to get the api provided by [denoland/deno_registry2](https://github.com/denoland/deno_registry2) (which gives you access to the original information in deno / x). The next best way is to get it from [nest.land](https://github.com/nestdotland/nest.land).

Would you like to develop deno-x-ranking together? 😄

**Comment by michael-spengler** [comment]
@yoshixmk If I'm informed properly the guys from nest.land are working on something that they call the module analyzer - also having some dynamic + static checks on the radar etc. 

We plan to facilitate regular TGIF Hackathons for nest.land - in this context I can imagine that we should bring the module analyzer + your ranking module together. Feel free to join the Deno Discord Server: https://discord.gg/nRnA5C

Tate will soon publish the Nest.land Discord Server ... 

... tbc ... :)

**Comment by t8** [comment]
That is correct! Keep us updated with this @yoshixmk. We'd love to see if there is an opportunity for us to collaborate.

**Comment by yoshixmk** [comment]
@michael-spengler @t8 Greate! I am also interested in Hackathons 👍 

I participated in Discord, but I didn't notice it.
I've changed it to look the same as my Github account on Discord now.

Has recruitment already started?

**Comment by t8** [comment]
Awesome to hear @yoshixmk! Recruitment hasn't started yet, but I'm sure @michael-spengler would be happy to keep you updated on this. Until then, feel free to join our Discord Server (where it'll be hosted!)

https://discord.gg/hYUsX3H

**Comment by yoshixmk** [comment]
I participated. Thank you!!
I am looking forward to your follow-up news 😄

**Comment by michael-spengler** [comment]
cool :) hackathon related updates will follow during the next days.

To connect some more dots I want to mention this one also as we can probably combine some cool things together: 
https://github.com/denoland/deno_registry2/issues/128#issuecomment-693685780

**Comment by michael-spengler** [comment]
Might be interesting: https://github.com/denosaurs/sauron
